operations/research-and-development/engineering/team/README.md

```diff
@@ -61,3 +40,32 @@ Below is how SET prioritizes what is worked on.
    * Pick up any bugs in triage without an obvious owning feature team.
    * Work on bugs in this order: hotfix, release, customer reported, other.
 4. [~Ask R&D](https://community-daily.mattermost.com/core/channels/ask-r-and-d) channel — this channel should be monitored, but checking it once or twice per day is sufficient.
+
+## Sources of escalations
+
+Work for SET comes from various sources:
+
+1. The SET [Kanban JIRA board](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=33). Tickets in the TO DO column should be organized from highest priority to lowest priority, based on the [Priority of Work](./#priority-of-work) section. The SET lead should make sure that the tickets are priority ordered but team members can move tickets around to meet the correct priorities as necessary. If tickets get stuck in a certain state, it's also the SET responsibility to push them to completion, either by nudging people or taking them over.
+2. OpsGenie: Support has an e-mail address they will send urgent escalations to. These result in pages in OpsDuty, first sent to the SET Primary, then to SET Back-up and ultimately to the SET Lead when not timely acknowledged.
+3. The [~Sustained Engineering](https://community-daily.mattermost.com/core/channels/sustained-engineering) channel. Support may use this channel to ask questions or raise urgent issues.
```

Comment:
I don't know if this is ordered by priority, but I've never had an OpsGenie alert and I've seen at least one or two posts in the channel from support during every rotation, which sometimes are left unanswered. In short: I feel like monitoring the channel is way more important than being worried about urgent alerts in OpsGenie.

Reply:
I just reordered them a little bit (next push). OpsGenie comes at the top, the truly time pressure items should go through that. Indeed, no such alert has happened for months, which is a good sign (or a sign that support forgot what the e-mail address is to use 😋 ). The most important thing to actively watch are the SET channel indeed.

Reply:
> OpsGenie comes at the top, the truly time pressure items should go through that.

I agree, but I have two problems with OpsGenie in general: I feel like I won't know what to do when (if) an alert comes in, as I've never done that, only in the 30-minute training meeting we had months (maybe a year or more?) ago when the SET on-call started. And I am not confident I'll get a notification from it (because that has never happened to me), so I'm constantly checking it just in case :tm: Those two things create a bit of stress and make me put the focus on the wrong part of the SET stack. Clarifying here that an alert from OpsGenie is rare would help, I think. 

> I agree with @agarciamontoro , people is afraid to call it an incident and sometimes they post a message in the channel. SET should be monitoring and pushing those to incidents without much fear.

Yeah, closing an unneeded incident is easy, but recovering the time lost when not opening one is hard. Creating incidents is cheap, so let's use them :)